src/ui/components/DialogShare/DialogShare.tsx

```diff
@@ -47,6 +48,11 @@ type DialogShareProps = {
     urlIdPrefix?: string;
     initialParams?: Record<string, number>;
     hasDefaultSize?: boolean;
+    showSelectorsCheckbox?: boolean;
+    showFederation?: boolean;
+    showEmbedLink?: boolean;
+    showHideMenu?: boolean;
+    showCopyAndExitBtn?: boolean;
```

Comment:
Let's rename `show` prefix to `with` prefix in these constants here

Reply:
Ok, also changed these variables:
```
    withLinkDescription?: boolean;
    withMarkupLink?: boolean;
    withHideComments?: boolean;
    withSelectorsCheckbox?: boolean;
    withFederation?: boolean;
    withEmbedLink?: boolean;
    withHideMenu?: boolean;
    withCopyAndExitBtn?: boolean;
```

Reply:
Let's remove `Checkbox` in `withSelectorsCheckbox`